confgenerator/confgenerator_test.go

```diff
@@ -30,7 +30,10 @@ import (
 const (
 	validTestdataDir   = "testdata/valid"
 	invalidTestdataDir = "testdata/invalid"
-	defaultConfig      = "default-config.yaml"
+	// Relative path to the ops-agent/confgenerator folder.
+	defaultUserConfPath = "default-config.yaml"
```

Comment:
[Optional] I wouldn't bother renaming this…

Reply:
The concern was that now we have split the built-in config and user specified config, `default-config.yaml` might not be descriptive enough.